Fix commands UTs

BSolonenko opened the pull request:
This PR is **ready** for review.

### Risk
This PR makes **no** API changes.

### Summary
This PR fixed unit test coverage for `get_urls_test.cc`, `sdl_activate_app_request_test.cc`, `alert_request_test.cc`,  `create_interaction_choice_set_test.cc`, `delete_interaction_choice_set_test.cc`, `on_button_notification_commands_test.cc`, `simple_response_commands_test.cc` and `subscribe_button_request_test.cc` units

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

BSolonenko commented:
@LitvinenkoIra  please review

LitvinenkoIra commented on src/components/application_manager/test/commands/hmi/get_urls_test.cc:
```diff
@@ -107,6 +107,7 @@ class GetUrlsTest : public CommandRequestTest<CommandsTestMocks::kIsNice> {
 
 TEST_F(GetUrlsTest, RUN_SUCCESS) {
   EXPECT_CALL(mock_policy_handler_, PolicyEnabled()).WillOnce(Return(true));
+  EXPECT_CALL(app_mngr_, GetRPCService()).WillOnce(ReturnRef(rpc_service_));
```
@BSolonenko please replace with ON_CALL (everywhere)
Try to avoid using EXPECT_CALLS for non critical for use cases getters.

BSolonenko replied:
@LitvinenkoIra  Fixed in https://github.com/LitvinenkoIra/sdl_core/pull/4/commits/976a6c79a2ae3df1626a91f07de3e0f80f8ec788

Commit:
Fix commands UTs

Fixed unit test coverage for
get_urls_test.cc,
sdl_activate_app_request_test.cc,
alert_request_test.cc,
create_interaction_choice_set_test.cc,
delete_interaction_choice_set_test.cc,
on_button_notification_commands_test.cc,
simple_response_commands_test.cc and
subscribe_button_request_test.cc units
